yarn.config.cjs

```diff
@@ -275,7 +275,7 @@ module.exports = defineConfig({
 
     // The list of files included in the package must only include files
     // generated during the build process.
-    workspace.set('files', ['dist', 'src/css/design-tokens.css']); // TODO remove src/css/design-tokens.css and add to build script
+    workspace.set('files', ['dist']);
```

Comment:
We can now align with the [metamask-module-template](https://github.com/MetaMask/metamask-module-template) workspace because everything is exported from dist